src/actions/projectMembers.js

```diff
@@ -85,6 +85,33 @@ export const fetchAllMembers = projectId => {
   };
 };
 
+/*
+ * Call API to find active members out of
+ * the members of one project
+ */
+export const getProjectActiveUser = () => {
+  const request = axios.get(ENDPOINTS.USER_PROFILES);
+  return async (dispatch, getState) => {
+    await dispatch(findUsersStart());
+    request
+      .then(res => {
+        let users = res.data;
+        let members = getState().projectMembers.members;
+        const memberList = [];
+        users = users.map(user => {
```

Comment:
Looks like `users` is not used again after this line. You could do `users.forEach` instead of `map`.
And then on line 98 `const users...` or just do `res.data.forEach...` right here on line 101.